src/main/java/com/catapult/app/example/services/CallbackServices.java

```diff
@@ -49,17 +45,69 @@ public synchronized void handleIncomingCallback(final String eventString, final
         try {
             Event event = EventBase.createEventFromString(eventString);
             if (event instanceof IncomingCallEvent) {
+                // Answer the incoming call
+                Call incomingCall = answerIncomingCall(event, userName);
 
-                if (event.getProperty("from") != null && sipPattern.matcher(event.getProperty("from")).find()) {
-                    // Case Incoming call from Endpoint to a PSTN number
-                    createCallFromEndpoint(event, userName, baseAppUrl);
-
-                } else if (event.getProperty("to") != null) {
-                    // Case Incoming call from PSTN to a number associated to an Endpoint
-                    createCallToEndpoint(event, userName, baseAppUrl);
+                Map<String, CallEvents> callEventMap = userEventCallMap.get(userName);
```

Comment:
I was stating something else. I was stating that member vars and local vars are both camel case with no rules the distinguish them.  For instance, I've seen member vars start with 'm' and local vars can start with anything.  

We don't need to address it now, but it is something to think about for later.